app/src/main/java/com/zeapo/pwdstore/ClipboardService.kt

```diff
@@ -45,12 +47,9 @@ class ClipboardService : Service() {
                 }
 
                 ACTION_START -> {
-                    val time = try {
-                        Integer.parseInt(settings.getString(PreferenceKeys.GENERAL_SHOW_TIME)
-                            ?: "45")
-                    } catch (e: NumberFormatException) {
-                        45
-                    }
+                    val time = runCatching {
+                        settings.getString(PreferenceKeys.GENERAL_SHOW_TIME)?.toIntOrNull()
+                    }.getOr(45)!!
```

Comment:
Does this really need catching? I think .toIntOrNull() ?: 45 should be good enough.

Reply:
Not really, no. I simply opted to retain the previous pattern of `try-catch`. Fixing this.

Reply:
Made the change here as well as a similar case in `BasePgpActivity` and squashed into individual commits for each file.